openedx/core/lib/block_structure/exceptions.py

```diff
@@ -15,3 +15,18 @@ class UsageKeyNotInBlockStructure(Exception):
     Exception for when a usage key is not found within a block structure.
     """
     pass
+
+
+class TransformerDataIncompatible(Exception):
+    """
+    Exception for when the version of a Transformer's data is not
+    compatible with the current version of the Transformer.
+    """
+    pass
+
+
+class BlockStructureNotFound(Exception):
+    """
+    Exception for when a Block Structure is not found.
+    """
+    pass
```

Comment:
Design question: Do we want to have a base `BlockStructureException` that these inherit from, to make it easier for external code to treat them uniformly?  (i. e., Something went wrong with block structures.  Bail on this.)